Sources/PrismCore/TemplateParser/TemplateParser+Token.swift

```diff
@@ -148,7 +148,8 @@ extension TemplateParser {
                 baseString = alignment
             case .textStyleLetterSpacing(let spacing):
                 if let spacing = spacing {
-                    baseString = "\(spacing)"
+                    let roundedSpacing = (spacing * 100).rounded(.toNearestOrEven) / 100
+                    baseString = "\(roundedSpacing)"
```

Comment:
Ok so I'd do this, at the bottom of this file

```swift
private extension Float {
    func roundToNearest() {
        (self * 100).rounded(.toNearestOrEven) / 100
    }
}
```

and then use this for spacing and line height. WDYT?

Reply:
I left it here because it used once. To reuse it for line heights, I'll move it to an extension.